onebusaway-android/src/main/res/values/strings.xml

```diff
@@ -568,6 +568,12 @@
         negative arrival times
     </string>
 
+    <string name="preferences_show_zoom_controls_title">Show zoom controls</string>
+    <string name="preferences_show_zoom_controls_summary">Display zoom controls on map
+    </string>
+
```

Comment:
Could you please remove the extra new lines here for consistency (just one line break)?